be/tool/main.c

```diff
@@ -32,12 +32,27 @@
 
 #include <stdio.h>              /* printf */
 #include <stdlib.h>             /* EXIT_FAILURE */
+#include <string.h>     	/* strcmp */
```

Comment:
The problem here is the "\tab" in the middle of this line.

In the middle of a line, we use "blank space" to make indentation. No tab.